### PR TITLE
Alex patch

### PR DESCRIPTION
**Pull request by z-alex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 - Sophia
 - Wei
-- Alex Zhang (z-alex)
+- Alex Zhang (z-alex)-
```
Comment (z-alex) [inline README.md]: test

File: soen.sql [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+-- -----------------------------------------------------
+-- Schema soen
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema soen
```
Comment (z-alex) [inline soen.sql]: Schema soen

File: soen.sql [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+-- -----------------------------------------------------
+-- Table `soen`.`rights`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `soen`.`rights` (
+  `id` INT(11) NOT NULL,
+  `name` VARCHAR(45) NOT NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
```
Comment (Helen2hang) [inline soen.sql]: bliblibli

File: .travis.yml [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
+
+services:
+  - mysql
+
+before_script:
+  - composer require phpunit/phpunit
+  - mysql -u root -e 'CREATE DATABASE soen;'
+  - mysql -u root soen < soen.sql
+  
+script: phpunit --configuration phpunit.xml --coverage-text
```
Comment (z-alex) [inline .travis.yml]: good

**Pull request merged**